content/gateway_guide.md

```diff
@@ -658,64 +722,15 @@ When your hot or cold wallet receives a payment whose purpose is unclear, we rec
 
 The first requirement to bouncing payments is [robustly monitoring for incoming payments](#robustly-monitoring-for-payments). You do not want to accidentally refund a user for more than they sent you! (This is particularly important if your bounce process is automated.) The [Partial Payment Flag Gateway Bulletin](https://ripple.com/files/GB-2014-06.pdf) explains how to avoid a common problem.
```

Comment:
Let's use the link to the KC for the partial payment GB: https://ripple.com/knowledge_center/partial-payment-flag/